mypyc/genops.py

```diff
@@ -1237,28 +1238,45 @@ def py_call(self,
                 line: int,
                 arg_kinds: Optional[List[int]] = None,
                 arg_names: Optional[List[Optional[str]]] = None) -> Value:
-        """Call to non-mypyc-generated function, e.g. some python builtin functions."""
-        # Box all arguments since we are invoking a non-mypyc-generated function.
-
+        """Use py_call_op or py_call_with_kwargs_op for function call."""
         if (arg_kinds is None) or all(kind == ARG_POS for kind in arg_kinds):
             return self.primitive_op(py_call_op, [function] + arg_values, line)
         else:
```

Comment:
Add a comment above the fastpath case, and then ditch the `else` and dedent the rest.